todo-list/src/App.js

```diff
@@ -0,0 +1,131 @@
+import { clear } from '@testing-library/user-event/dist/clear'
+import React, {useState, useEffect} from 'react'
+import "./App.css"
+import Todo from './Component/Todo'
+import Todolist from './Component/Todolist'
+
+
+
+const App = () => {
+  const [todos, setTodos] = useState([])
+  const LOCAL_STORAGE_KEY= 'todoList.app'
+  const [status,setStatus] = useState("all")
+  const [filteredTodos, setFilteredTodos] = useState([])
+
+    //check if the input is already existed in the local storage
+    
+    //addTodo
+    const addTodo = (text) => {
+      let id = 1;
+      if(todos.length > 0) {
+        id = todos[0].id + 1
+      }
```

Comment:
Will duplicate id, please use other methods, You can use `Date.now()` or random string.